### ISSUE TITLE
graphviz error (conda conflicts) after sphinx 1.6.5 update

### ISSUE THREAD

**Issue by cokelaer** [rtfd/readthedocs.org#3197]
## Details

* Project URL:  http://readthedocs.org/projects/bioconvert
* Build URL (if applicable):  http://readthedocs.org/projects/bioconvert/builds/6180152/
* Read the Docs username (if applicable): bioconvert.readthedocs.io

## Expected Result

succesful build

## Actual Result

The build of the sphinx gallery fails because the graphviz/pygraphviz installation is broken.

When using the environment_rtd.yaml, we install the correct graphviz/pygraphviz installation by pinning the iuc dependency to icu=56 (instead of icu=58) because graphviz dependencies libgvplugin_pango.so.6 not found (https://github.com/ContinuumIO/anaconda-issues/issues/1357)

This works fine locally. However, on readthedocs, the installation of sphinx, upgrade icu56 to icu58 hence breaking the graphviz package. 

I do not see any solution but the built worked last week. So, I suspect that sphinx 1.6.5 force the installation of icu58 for some reasons. 

I do not see any solution at the moment. If you have an idea, I can try. thanks

**Comment by cokelaer** [comment]
More investigation shows that the issue could be with pillow. In the second line of the build, 
```
 conda install --yes --name master mock pillow sphinx sphinx_rtd_theme 
```
pillow forces the installation of icu=58 as a dependency of  pillow=4.3.0

**Comment by cokelaer** [comment]
So one temporary solution could be to pin down icu to version 56 but this would mean an extra conda command after sphinx and others are installed. 

Won't it be possible to skip this step:

```
conda install --yes --name master mock pillow sphinx sphinx_rtd_theme
```
and let users deal with the conda installation ? thanks

**Comment by netflixhelp12** [comment]
Netflix Ui 122 Ps4 | Netflix support | How To Install Netflix On Ps4 | Playstation 4 Netflix Problems | Ps4 Netflix Problems Today | Netflix On Ps4 Issues | http://netflix-netflixsupport.com | Netflix Nw-2-5 | Netflix Down | Netflix Error Ui-122 Ps4 | Netflix Error Ui-122 Ps3 | Netflix Error Code Ui-122 Ps4 | Netflix Code Ui-122 Ps3 | Netflix Error Code Ui-122 On Ps3 http://netflix-netflixsupport.com

You’re facing any Technical problem for Netflix.We provide you Netflix Account Support Service 1844-777-7886 by Phone Call Toll Free Customer Care Support Contact Number. http://netflix-netflixsupport.com

**Comment by agjohnson** [comment]
Thanks for poking around with this. We can't skip the step where we install our base requirements, but we've unpinned our dependencies to make forcing dependencies easier. Does installing the version of pillow that you require in your environment help?

**Comment by cokelaer** [comment]
It does not seem to fix the issue if I include pillow in the environment.yml file. Indeed the version installed zith icu=56 is pillow 4.0 . Therefore the next command "conda install pillow ...." update pillow to 4.3 hence icu to v58. I do not think there is any solution right now unfortuantely.Probably a conda/conda-forge issue to be reported. You can close this issue if you think it makes sense. thanks

**Comment by stsewd** [comment]
@cokelaer Some of your builds are passing, were you able to do a workaround for this?

**Comment by stsewd** [comment]
Closing this as the user [hasn't responded or given more information](https://docs.readthedocs.io/en/latest/contribute.html#initial-triage), feel free to reopen with more information if you are still having this problem :).

**Comment by cokelaer** [comment]
FYI, I have created a singularity container that contains the dot executable. It is available here: https://www.singularity-hub.org/collections/286 

Then, when building the documentation, the Python code that is created on the fly within the sphinx build will download and use the graphiz container. That way, I am bypassing the need to install pygraphivz/graphviz which was causing the issue. 

So, I have solve my issue but this is a hack for now. Maybe this has been solved in recent version of RTD/sphinx /conda combo.
